app/routes/dashboard.apps.$appId.security/view.tsx

```diff
@@ -298,7 +533,7 @@ export const SecurityView = ({ endpoint, appId, blockchains }: SecurityViewProps
               aria-label={t.security.contractAria}
               size="xs"
               type="button"
-              variant="filled"
+              variant="outline"
               onClick={() => {
                 if (whitelistContractsInput === "" || whitelistContractsDropdown === "") {
```

Comment:
I know this is an old code, but since we're touching the file, I think this logic is big enough to be placed in a function instead of writing all of this within JSX